validations/src/validations.rs

```diff
@@ -499,6 +508,16 @@ pub fn validate_data_request_output(request: &DataRequestOutput) -> Result<(), T
         });
     }
 
+    if active_wips.wip0022() {
+        let reward_collateral_ratio = request.collateral / request.witness_reward;
```

Comment:
Not sure if this handles the case where `collateral` is set to 0 correctly, it should be interpreted as `collateral_minimum`.

Reply:
It indeed does not, I assumed you could never receive one with that setting here since other parts of the code would already have set it to `collateral_minimum`. Not a big problem to add it though, just to play if safe.

Reply:
Done.